doc/source/commands/runprogram.rst

```diff
@@ -0,0 +1,226 @@
+.. _running:
+
+Running Programs
+================
+
+.. contents:: Contents
+    :local:
+    :depth: 2
+
+KerboScript supports running executing code saved in files on the local or
+archive disks. The terms "program" and "script" are generally used
+interchangably when describing this behavior. Programs can either be run using
+one of the :ref:`runfunctions` or the :ref:`run`.  Supported file types are:
```

Comment:
When you compile the docs these references expand to match the referenced heading.  So `:ref:'run'` will turn into a link reading "Run Keyword".  It's just sphinx/rest shorthand.